cola/gitcmds.py

```diff
@@ -658,6 +658,28 @@ def diff_upstream(context, head):
     return diff_filenames(context, base, tracked)
 
 
+def list_submodule(context):
+    """Return submodules in the format(state, sha1, path, describe)"""
+    git = context.git
+    SHA1_LENGTH = 40
```

Comment:
Instead of `SHA1_LENGTH`, please use the existing `OID_LENGTH` which is already imported in this module.

Reply:
Done. I am wondering whether the outputs of git is stable across different versions.